install_pim/system_requirements/system_requirements.rst.inc

```diff
@@ -22,25 +22,37 @@ We can only guarantee the behaviour on the following web server:
 
 The web server will also need the following libraries and modules:
 
-+--------------+-----------------------------------------+
-| mod rewrite  |  Required                               |
-+--------------+-----------------------------------------+
-| mod php5     |  Required (no CGI, no FastCGI, nor FPM) |
-+--------------+-----------------------------------------+
++--------------+------------------------------------------------+
+| mod rewrite  |  Required                                      |
++--------------+------------------------------------------------+
+| php7.1 fpm   |  Required (no CGI, no FastCGI, nor libmod-php) |
```

Comment:
Conflict with what is written here: https://github.com/akeneo/pim-docs/pull/611/files#diff-f7f42f644ed8a84bc7faffdd561285a1R75